implementations/c/include/ockam/error.h

```diff
@@ -21,5 +21,6 @@ typedef uint32_t ockam_error_t;
 #define OCKAM_ERROR_INTERFACE_TRANSPORT    (4u << OCKAM_ERROR_INTERFACE_SHIFT)
 #define OCKAM_ERROR_INTERFACE_KEYAGREEMENT (5u << OCKAM_ERROR_INTERFACE_SHIFT)
 #define OCKAM_ERROR_INTERFACE_RANDOM       (6u << OCKAM_ERROR_INTERFACE_SHIFT)
+#define OCKAM_ERROR_INTERFACE_IO           (6u << OCKAM_ERROR_INTERFACE_SHIFT)
```

Comment:
The interface ids should't conflict. IO should be bumped to 7.

Reply:
Good catch, I'll fix it.